client/src/modules/list/components/Items/InputBar/InputBar.jsx

```diff
@@ -36,6 +39,10 @@ class InputBar extends Component {
     if (isTipVisible) {
       this.hideTipAfterTimeout();
     }
+
+    if (!this.socket) {
+      this.socket = io(FULL_URL);
```

Comment:
why you delete namespace '/sack'? Can we use rooms without namespaces?

Reply:
yes we can.
You said yesterday that they're attached to global namespace